src/plots/cartesian/layout_attributes.js

```diff
@@ -680,6 +680,15 @@ module.exports = {
             'If *B*, 1B.'
         ].join(' ')
     },
+    minexponent: {
+        valType: 'number',
+        dflt: 3,
```

Comment:
What is the minimum value for this parameter?
You could declare that using `min: ...`  here.

Reply:
Thanks, added in 83273cf54ab951d745afe0f1ae272ed3ad460ecc